model/registry/instrumentation.yaml

```diff
@@ -0,0 +1,24 @@
+groups:
+- id: registry.instrumentation
+  prefix: instrumentation
+  type: attribute_group
+  brief: >
+    Describes properties about the library that is being instrumented or that
+    is responsible for performing the instrumentation.
+  attributes:
+    - id: source.repo
+      type: string
+      stability: experimental
+      brief: The repository associated with the InstrumentationScope.
+      examples: [github.com/organization/repo, gitlab.com/group/repo]
+    - id: source.target
+      type: string
+      stability: experimental
+      brief: The build target associated with the InstrumentationScope.
```

Comment:
```suggestion
      brief: The instrumentation library artifact name
```

Reply:
To me, "artifact" implies that it names the output/result; while "target" implies the structure of the build in the source. The problem with the "artifact" nomenclature is that it requires that the project has reached a sufficient level of maturity as to have packaging in place, precluding complete instrumentation before that...

I think it would be ideal to be able to track and differentiate these components prior to packaging/deployment being completed. It might be nice to have additional attributes available related to packaging/deployment, however, it would be good to be able to begin tracking / differentiating at an earlier lifecycle stage.

Reply:
Given this feedback, I added some distribution-related attributes in:

 - https://github.com/open-telemetry/semantic-conventions/pull/1094

Let's continue that conversation there.